### ISSUE TITLE
Persona POC - Base model evaluation - friends fan persona

### ISSUE THREAD

**Issue by advaithsrao** [vanshigupta04/Persona-LLM-Chatbot-For-QA#4]
For each persona we validate the base model on MCQ evaluation first, to see if fine-tuning is needed or not for the persona

put the base LLM against the dataset and see how well it does. 
  a. If accuracy < 40%, then we can go ahead with Finetuning the model for our persona. 
  b. Elif try different model fm466@scarletmail.rutgers.edu; task 3 repeats
  c. Else: drop persona: task 1 repeats

**Comment by advaithsrao** [comment]
MCQ Evaluation function - This should cover a function that does the following,
1. Prompt engineering -> Few shot for answer formatting
2. Measure evaluation 2:  Prompt engg vs non-prompt (both w/ fine-tuned model))  
3. Get prediction for the LLM on the dataset provided
4. Return accuracy

**Comment by advaithsrao** [comment]
This issue is to look at which personas are worth working on. If the base model has an accuracy of more than 40%, we prefer not adding the persona to our fine-tuning suite

**Comment by advaithsrao** [comment]
@vanshigupta04
